lib/helpers.js

```diff
@@ -0,0 +1,303 @@
+require('../config/env');
+
+const YAML = require('js-yaml');
+const fs = require('fs');
+const glob = require('glob');
+const _ = require('lodash');
+const nconf = require('nconf');
+const request = require('request');
+const async = require('async');
+const path = require('path');
```

Comment:
`npm run test` fails on this line 

```
  10:7  error  'path' is assigned a value but never used  no-unused-vars
```

Reply:
fixed (leftover)
if you want to run it you need to run new VK with linked new HH, run apiary  and change /uncomment 
https://github.com/apiaryio/constitutions/blob/tu1ly/refactor/config/env.js#L17
https://github.com/apiaryio/constitutions/blob/tu1ly/refactor/config/env.js#L15